lib/tasks/prescat.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :prescat do
+  desc 'Migrate storage root, returning druids of all migrated moabs'
+  task :processing, [:from, :to] => :environment do |_task, _args|
+    migration_service = StorageRootMigrationService.new(args(:from), args(:to))
```

Comment:
i copypasta'ed this as boilerplate for a rake task i was writing, and noticed that you probably actually want `args[:from]` and `args[:to]`, as opposed to the parens version here.  or at least, the square brackets approach worked for me, whereas the parens didn't.

Reply:
Thanks. Fixed.